tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -671,8 +670,19 @@ docker:
 EOT
 	fi
 
+	if [ "${UMOCI}" = "yes" ]; then
+		umoci_url="$(get_package_version_from_kata_yaml externals.umoci.url)"
+		umoci_tag="$(get_package_version_from_kata_yaml externals.umoci.tag)"
+		info "Install umoci"
+		git clone "${umoci_url}" --branch "${umoci_tag}"
+		pushd umoci
+		make
+		install -o root -g root -m 0755 umoci "${ROOTFS_DIR}/usr/local/bin/"
+		popd
+	fi
+
 	if [ -n "${AA_KBC}" ]; then
-		[ -z "${SKOPEO_UMOCI}" ] && die "SKOPEO_UMOCI must be set to install attestation-agent"
+		[ -z "${UMOCI}" ] && die "UMOCI must be set to install attestation-agent"
```

Comment:
Shouldn't this be testing for `"yes"` explicitly?

Reply:
Good point. It should be updated now. Thanks!